frontend/package.json

```diff
@@ -54,7 +55,8 @@
     "eject": "react-scripts eject",
     "lint": "eslint src/ --ext .ts,.tsx",
     "lint-fix": "eslint src/ --ext .ts,.tsx --fix",
-    "prettier": "prettier --write src/**/*.ts src/**/*.tsx"
+    "prettier": "prettier --write src/**/*.ts src/**/*.tsx",
+    "update-api-client": "./scripts/update-api-client.sh"
```

Comment:
@xitij2000 Handy to auto update API client.

Reply:
@giovannicimolin  Could we have a step in the CI try to auto-generate the api-client and if that modifies files we can fail the deployment? This will make sure the API client is always up to date.